docker/Dockerfile

```diff
@@ -1,9 +1,16 @@
-FROM alpine:latest as alpine
- 
+FROM --platform=$TARGETPLATFORM alpine:latest as alpine
+ARG TARGETPLATFORM
 RUN apk add -U --no-cache ca-certificates
- 
+RUN mkdir -p /linux/amd64
+ADD ./linux_amd64/release /linux/amd64
```

Comment:
Should this be `ADD ./dist/linux_amd64/release /linux/amd64` ? - because the binaries will be found in that path according to our [documentation](https://github.com/dapr/dapr/blob/master/docs/development/developing-dapr.md#build-the-dapr-binaries)

Reply:
No. the command line that we use in makefile specifies ./dist as a docker root. so we should not add ./dist

$(DOCKER) buildx build --platform $(DOCKERMUTI_ARCH) -t $(DOCKER_IMAGE_LATEST_TAG) $(OUT_DIR)